packages/storybook/src/docs/component-apis.mdx

```diff
@@ -45,7 +45,7 @@ Or in a named slot:
 ## Attributes and Properties
```

Comment:
I will wait for #2131 to go in before merging this.

Also conflicts with #2111 but we'll see which is ready first.